.pre-commit-config.yaml

```diff
@@ -10,7 +10,7 @@ repos:
 
   # Python formatter
   - repo: https://github.com/psf/black
-    rev: 22.6.0
+    rev: 22.10.0
```

Comment:
updating black to recent version. doesn't affect the formatting at all (updates never should with this formatter)